testsuite/gateway/gateway_api/route.py

```diff
@@ -43,6 +44,21 @@ def create_instance(
 
         return cls(model, context=openshift.context)
 
+    def is_affected_by(self, policy: Policy):
+        """Returns True, if affected by status is found within the object for the specific policy"""
+        for condition_set in self.model.status.parents:
+            if condition_set.controllerName == "kuadrant.io/policy-controller":
+                for condition in condition_set.conditions:
+                    if check_condition(
+                        condition,
+                        f"kuadrant.io/{policy.kind(lowercase=False)}Affected",
```

Comment:
is it intended that there is no space between object reference and word Affected?

Reply:
It is, the final name looks like this `kuadrant.io/RateLimitPolicyAffected`